SCClassLibrary/Common/Control/Server.sc

```diff
@@ -267,6 +267,7 @@ Server {
 
 	var <name, <addr, <clientID, <userSpecifiedClientID = false;
 	var <isLocal, <inProcess, <>sendQuit, <>remoteControlled;
+	var numClients; // maxLogins as sent from booted scsynth
```

Comment:
why not call it `maxLogins`?  Hm, I understand that `clientID` and `numClients` are a pair.

Reply:
I find that maxLogins is a only technical name, numClients is more semantic: 
it is the number of clients that the server is prepared to handle.

Reply:
well, you could read it as the number of clients the server currently has.